zebrium/README.md

```diff
@@ -4,41 +4,10 @@
  
 When you know there's a problem and you're not sure what caused it, [Zebrium][1] shows you the root cause directly on your Datadog dashboards. It works by using machine learning on logs, without the need for manual training or setting up any rules, and it achieves accuracy in less than 24 hours. 
 
-Using Zebrium is simple. When troubleshooting a problem, simply add the Zebrium app to your Datadog dashboard and look at the details of the corresponding detection.
-
-Zebrium integrates with Datadog in two ways: 1) a Datadog App with a custom dashboard widget and 2) an events and metrics integration.
-
-### 1) Datadog App
-
-The Zebrium app provides a pre-built, interactive dashboard widget that displays the detected problems over time, and allows you to dig into the Root Cause of these problems (as well as relevant metrics). This method provides the simplest user experience.
- 
-### 2) Events and metrics integration
-
-With the integration, Zebrium detection events and metrics are sent directly into Datadog. You can visualize them any way you'd like (sample dashboard provided). This method should be used if you would like to customize how Zebrium data appears on your dashboards.
```

Comment:
Can we keep this portion of it 'Zebrium detection events and metrics are sent directly to Datadog.'